src/ring_theory/power_series/basic.lean

```diff
@@ -1196,6 +1196,23 @@ rescale (-1 : A)
 @[simp] lemma eval_neg_hom_X : eval_neg_hom (X : power_series A) = -X :=
 rescale_neg_one_X
 
+/-- Two ways of truncating a power series are the same. -/
```

Comment:
You have this lemma in the "comm_ring" section - is it true for `ring`?

Reply:
It is true for `ring`. Of course the proof of the second lemma (using commutativity and the first lemma) won't fly then. 
I moved it to the `ring` section and modified the proof of the second lemma such that it works (similar to what is done with the first lemma. Luck has it that #6370 providing `coeff_succ_X_mul` just landed...